cmd/kots/cli/install.go

```diff
@@ -446,7 +451,13 @@ func InstallCmd() *cobra.Command {
 					log.ActionWithSpinner("Waiting for preflight checks to complete")
 					if err := ValidatePreflightStatus(deployOptions, authSlug, apiEndpoint); err != nil {
 						log.FinishSpinnerWithError()
-						return errors.Wrap(err, "failed to validate preflight results")
+						log.Errorf("Preflight checks contain warnings or errors. The application was not deployed")
+						perr := preflightError{}
+						if errors.As(err, &perr) {
+							print.PreflightErrors(log, perr.Results)
+							cmd.SilenceErrors = true // Stop Cobra from printing the error, we print it ourselves
```

Comment:
instead of silencing the errors, can we just return/print a generic error message?

Reply:
> instead of silencing the errors, can we just return/print a generic error message?

That extra line after the formatted preflight check results at the end did not feel right. This is how it ends up looking like

```sh
evans@evans-vm3:~$ ./kots install thanos-reloaded -n default --shared-password password --license-file ~/thanos_license.yaml --kubeconfig /etc/rancher/k3s/k3s.yaml
  • Deploying Admin Console
    • Creating namespace ✓
    • Waiting for datastore to be ready ✓
  • Waiting for Admin Console to be ready ✓
  • Waiting for installation to complete ✓
  • Waiting for preflight checks to complete ✓
  • Preflight checks contain warnings or errors. The application was not deployed

STATE    TITLE                          MESSAGE
PASS     Required Kubernetes Version    Your cluster meets the recommended and required versions of Kubernetes.
PASS     Container Runtime              containerd container runtime was found.
WARN     Kubernetes Distribution        Unable to determine the distribution of Kubernetes.
WARN     Total CPU Cores                The cluster should contain at least 4 cores.

Error: There are preflight check warnings for the application. The app was not deployed.
```

NOTE: Any other errors will get printed, we just silence the errors when we format the preflight checks table since we are printing our own error message.

Reply:
Here it is as an image to show the colours

<img width="1181" alt="image" src="https://github.com/replicatedhq/kots/assets/681087/21168752-2233-4c87-bbe4-60194aaa7698">